include/Dialect/BGV/Transforms/Passes.td

```diff
@@ -0,0 +1,14 @@
+#ifndef INCLUDE_DIALECT_BGV_TRANSFORMS_PASSES_TD_
+#define INCLUDE_DIALECT_BGV_TRANSFORMS_PASSES_TD_
+
+include "mlir/Pass/PassBase.td"
+
+def AddClientInterface : Pass<"bgv-add-client-interface"> {
+  let summary = "...";
+  let description = [{
+  ...
```

Comment:
nit: could you add just a few words here?

Reply:
ah, thanks for catching that!

Reply:
Fixed